packs/data/feats.db/rolling-landing.json

```diff
@@ -0,0 +1,39 @@
+{
+    "_id": "bJ4tF95Byy7wCS65",
+    "img": "systems/pf2e/icons/default-icons/feats.webp",
+    "name": "Rolling Landing",
+    "system": {
+        "actionType": {
+            "value": "passive"
+        },
+        "actions": {
+            "value": null
+        },
+        "category": "skill",
+        "description": {
+            "value": "<p>You land with quick rolls that help you keep your momentum. If you fall and don't take damage (usually due to treating the fall as a shorter distance), you can use your reaction to immediately enter a short roll when you land and Step. If you are an expert in Acrobatics, you can use your reaction to Step or Stride up to half your Speed. If you're a master in Acrobatics, you can use your reaction to Step or Stride up to your full Speed. If you're legendary in Acrobatics, you don't trigger reactions triggered by movement when you Stride in this way.</p>"
+        },
+        "level": {
+            "value": 2
+        },
+        "prerequisites": {
+            "value": [
+                {
+                    "value": "Cat Fall"
+                },
+                {
+                    "value": "trained in Acrobatics"
+                }
+            ]
+        },
+        "rules": [],
+        "source": {
+            "value": "Pathfinder Lost Omens: Firebrands"
+        },
+        "traits": {
+            "rarity": "common",
+            "value": []
```

Comment:
Is this missing traits?

Reply:
nope, general and skill are set automatically by the system.